core/tests/protractor.conf.js

```diff
@@ -338,6 +339,27 @@ exports.config = {
       }));
     }
 
+    // Adding a video reporter. For more information see
+    // https://www.npmjs.com/package/protractor-video-reporter
+
+    var ffmpegArguments = [
+      '-y',
+      '-r', '30',
+      '-f', 'x11grab',
+      '-s', '1366x768',
+      '-i', process.env.DISPLAY,
+      '-g', '300',
+      '-vcodec', 'qtrle',
+    ]
+
+    jasmine.getEnv().addReporter(new VideoReporter({
+      baseDirectory: path.resolve(__dirname, '../../../protractor-video'),
+      singleVideo: false,
+      singleVideoPath: 'uuid',
```

Comment:
Do we need a better name for the video files to identify which video belongs to which suite instead of random id?

Reply:
This would be ideal, but we have been running into issues getting it to work (see earlier discussion in this PR). We're using UUIDs for now to help with debugging and will improve the naming later if it turns out to be an issue